api_tests/lib_sdk/actors/actor.ts

```diff
@@ -3,17 +3,16 @@ import { BigNumber, Cnd, ComitClient, Swap } from "comit-sdk";
 import { parseEther } from "ethers/utils";
 import getPort from "get-port";
 import { Logger } from "log4js";
-import { E2ETestActorConfig } from "../../lib/config";
-import { LedgerConfig } from "../../lib/ledger_runner";
-import "../../lib/setup_chai";
+import { E2ETestActorConfig } from "../ledgers/config";
```

Comment:
Should `E2ETestActorConfig` really be in `./ledgers/config` ? I'd see it more somewhere in the actor module. Or maybe we need to rename?